src/main.cpp

```diff
@@ -4727,10 +4727,11 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             RelayTransaction(tx);
             vWorkQueue.push_back(inv.hash);
 
-            LogPrint("mempool", "AcceptToMemoryPool: peer=%d: accepted %s (poolsz %u txn, %u kB)\n",
+            LogPrint("mempool", "AcceptToMemoryPool: peer=%d: accepted %s (poolsz %u txn, %u MiB, cache=%.1fMiB(%utx))\n",
```

Comment:
I thought we were parsing the maxmempool size in `MB` so it should be changed to `kB` in the other places instead.